src/options.ts

```diff
@@ -146,7 +146,7 @@ const inferOptions = (inputs: Inputs): Options => {
   // if micromambaUrl is specified, use that, otherwise use micromambaVersion (or 'latest' if not specified)
   const micromambaSource = inputs.micromambaUrl
     ? right(inputs.micromambaUrl)
-    : left(inputs.micromambaVersion || 'latest')
+    : left(inputs.micromambaVersion || '1.5.10-0')
```

Comment:
this would require updating the GHA each time a 1.x is released, which I find fragile, but I don't have a better solution for now.

Reply:
every other solution would imply some weird api calls to get the latest v1 version which could also break